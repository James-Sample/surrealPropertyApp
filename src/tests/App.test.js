import React from "react";
import { render, screen } from "@testing-library/react";
import App from "../components/App";
import { BrowserRouter } from "react-router-dom";

describe("it renders the app", () => {
  xit("renders the app", () => {
    const { asFragment } = render(
      <BrowserRouter>
        <App />
      </BrowserRouter>
    );
    expect(asFragment()).toMatchSnapshot();
  });

  xit("renders the Title", () => {
    render(
      <BrowserRouter>
        <App />
      </BrowserRouter>
    );
    const linkElement = screen.getByText(/Surreal Estate/i);
    expect(linkElement).toBeInTheDocument();
  });
});
