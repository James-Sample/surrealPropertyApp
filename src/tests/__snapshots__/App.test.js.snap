// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders the app renders the app 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div>
      <nav
        class="navBar"
      >
        <img
          alt="housepic"
          class="logoPic"
          src="/HousePic.png"
        />
        <h2
          class="title"
        >
          Surreal Estate
        </h2>
        <ul
          class="navbar-links"
        >
          <li
            class="navbar-links-item"
          >
            <a
              href="/"
            >
              View Properties
            </a>
          </li>
          <li
            class="navbar-links-item"
          >
            <a
              href="/add-property"
            >
              Add a Property
            </a>
          </li>
        </ul>
      </nav>
    </div>
    <div>
      Properties page
    </div>
  </div>
</DocumentFragment>
`;
